tests/scripts/basic-build-test.sh

```diff
@@ -150,123 +156,139 @@ echo "========================================================================="
 echo "Test Report Summary"
 echo
```

Comment:
Lines 155-157 look like a perfect header for the coverage-summary.txt file.

Apart from that LGTM.